functions/controllers/user.js

```diff
@@ -0,0 +1,76 @@
+const { db, admin } = require("../util/admin");
+const config = require("../util/config");
+const firebase = require("firebase");
+firebase.initializeApp(config);
+
+const { validateSignUpData } = require("../util/validator");
+
+exports.signupUser = (req, res) => {
+  const newUser = {
+    fullname: req.body.fullname,
+    username: req.body.username,
+    email: req.body.email,
+    number: req.body.number,
+    password: req.body.password,
+    confirmPassword: req.body.confirmPassword,
+    userstatus: req.body.userstatus
+  };
```

Comment:
you want to destructure this.
```
const = {fullname, username, email, number, password, confirmPassword, userStatus} = req.body;
```
avoid repetition of codes.
practice camelCase and be consistent. e.g `userstatus` should be `userStatus`